app/controllers/products_controller.rb

```diff
@@ -0,0 +1,40 @@
+class ProductsController < ApplicationController
+  def new
+    @header = "Add A New Product"
+    @product = Product.new
+    @action = "create"
+    @method = :post
+  end
```

Comment:
Watch your whitespace in this class - there should be one line of whitespace between each method defitinition